.mvn/maven.config

```diff
@@ -1,2 +1 @@
 --strict-checksums
-
```

Comment:
Can you move the first commit to a separate PR to get it merged quicker?

Reply:
It's related

Reply:
It doesn't matter how fast it'll get merged